src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -312,11 +399,8 @@ void SettingsLoader::FindFragmentsAndMergeIntoUserSettings()
 
     for (const auto& ext : extensions)
     {
-        const auto packageName = ext.Package().Id().FamilyName();
-        if (_ignoredNamespaces.contains(std::wstring_view{ packageName }))

```

Comment:
I would still prefer to bail out early--before interacting with any more of the async package APIs (!)--if we _aren't_ expressly building the fragment data and the user has requested that we please disable it. Is that still possible?